City-of-Helsinki/berth-reservations
# VEN-1024 | No berth confirmation email

## Description :sparkles:
* Refactor `applications/notifications.py` as a separate sub-module
* Test that the `no berth` email has been sent
* Remove the unused `NO_SUITABLE_BERTH_NOTIFIED`, assuming that once it's been set to that status, the customer will ALWAYS be notified

## Issues :bug:
### Closes :no_good_woman:
**[VEN-1024](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1024):**  Send No Berth email

## Testing :alembic:
### Automated tests :gear:️
```shell
$ pytest applications/tests/test_applications_new_schema_mutations.py::test_reject_berth_application
```

### Manual testing :construction_worker_man:
1. Send the `rejectBerthApplication` mutation
2. You should see on the console the rejection email for that application

## Additional notes :spiral_notepad:
The confirmation email was already being sent, the new assertions check for that
